frontend/src/helpers/app/generate-telegram-link.helper.ts

```diff
@@ -0,0 +1,7 @@
+const TELEGRAM_URL = 'https://t.me/';
+
+const generateTelegramLink = (username: string): string => {
+  return TELEGRAM_URL + username;
```

Comment:
Maybe it'd be better to make concatenation by template strings - just to avoid misleading of using `+` for concatenation
```suggestion
  return `${TELEGRAM_URL}${username}`;
  // or just
  return `https://t.me/${username}`;
```